### ISSUE TITLE
Snackbar: let's perhaps use it as a notification card instead…

### ISSUE THREAD

**Issue by Kiarokh** [Lundalogik/lime-elements#2027]
## Background information
Classically, the snackbars pop up at the bottom of the screen. They are typically dark on light background, so it's easy to notice them. But with the dark mode changes, we kind of ended up with a light snackbar on a light background, as a natural result of what we are doing.
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/154725/212075845-19993e04-4b42-4378-a68d-f8dcbce027f5.png">

### 1. They must be easily noticed by the user
From accessibility point of view, when a Snackbar is shown, we kind of need something that sticks out visually. So it's easy to notice the message. Particularly because it pops-up at the bottom of the screen, where users do not look. I don't have the energy to find the reference and link it here, but homo sapiens (us) are particularly good at noticing small changes in the left and right sides of the peripherals of our vision. That's where lions or hyenas (or other sapiens) would suddenly pop-up to hunt us, and not seeing those types of dangers would mean the genes would get our of the gene-pool. This is why games have such vital data, maps, life indicators, etc… on those areas normally. Also why notifications on wide screens pop up on those areas.

We use snackbars to inform users about changes that require attention. They are kind of "notifications". 

So I'd argue that's a better place for them is on the top-right corner of the screen. This way, we can keep the light color, even in light mode. No reason to invert the colors really.

### 2. Assistive tech should immediately read them out loud
Snackbars (if used correctly) typically show warnings or inform the users about some important message. A sighted user sees them hopefully. But a user that does not see, relies on the screen reader to understand something has happened. Currently the snackbar component has `role="status"` and any time its shown, the screen reader will read out its contents ✅ So we don't have to do anything.

### 3. Sighted users won't see multiple snackbars!
However, all the snackbars are stacked on top of each other. This makes it impossible for sigthed users to see them all.
Since they have a timeout, and disappear after a while, users won't know what message was there before. So multiple snackbars are not accessible for sighted users as of today.

### 1. Dismissiblity
I believe all snackbars should have the dismiss button ON by default. Because they cover the content on the screen, users should be able to dismiss them. Specially when we move them to the top-right corner of the screen. So `dismissible ` should be `true` by default, today it's `false`.

### 2. I have no time to read
Snackbars are only there for a very short time. Sometimes users don't have enough time to read the content; either because they are dyslexic or because the content is too large and the developers didn't think about increasing the `timeout`. There are two problems here:
1. There is no visual indication that there is a timeout and the snackbar will close soon
2. there is no way for the end user to pause this closing action, for example by clicking a ⏸️  button on the snackbar or by hovering it.

We need to address these issues.

**Comment by Kiarokh** [comment]
moved here: https://github.com/Lundalogik/crm-feature/issues/3349
